V1Michigan/v1
# Add slack link, Addresses PLA-16 (in Linear)

### Status
**READY**

### Description
Slack community added on Platform.

### Related PRs
None


### Deploy Notes
None. New Tables constructed to keep track of who joins. 

### Dependencies
No changes. 

### Migrations
NO

### Steps to Test or Reproduce
Don't use netlify preview link. Those have not been accounted for in the db access whitelist in Supabse. Use localhost:3000 following the directions below this paragraph. To see the actual change. Log in, head to dashboard and click v1 slack. Check Db tables in Supabase to see if it updated. See what happens if link gone to without being logged in. 

```sh
git pull 
git checkout add-slack-link
npm run dev
```

1. 

### Impacted Areas in Application
List general components of the application that this PR will affect:

Dashboard, Added new Community Page
======================
Closes PLA-16
